gitpod-io/website
# chore: Add custom docker image requirements

## Description


Adds a new section in the docs on configuring a custom Dockerfile with a non-gitpod base image.

## Related Issue(s)

Fixes #2850
Fixes https://github.com/gitpod-io/website/issues/790

## How to test


Go to `<workspace-url>/docs/configure/workspaces/workspace-image#custom-base-image` and see the new docs section.

## Release Notes

```release-note
New section on configuring a custom Dockerfile with non-gitpod base image
```

## Documentation



<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2886"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2886"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
